packages/preview/hsrm-internship-report/0.0.1/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "hsrm-internship-report"
```

Comment:
Because this is not an official template, another name would be more appropriate. [We have specific naming guidelines](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#naming-rules).